ieee.csl

```diff
@@ -214,20 +225,27 @@
         <text variable="DOI" prefix="doi: "/>
       </else-if>
       <else>
-        <group delimiter=". ">
+        <group delimiter=". " suffix=". ">
           <group delimiter=": ">
             <text term="accessed" text-case="capitalize-first"/>
             <date variable="accessed">
-              <date-part name="month" form="short" suffix=" "/>
+              <date-part name="month" form="short" suffix=". "/>
               <date-part name="day" form="numeric-leading-zeros" suffix=", "/>
               <date-part name="year"/>
             </date>
           </group>
-          <text term="online" prefix="[" suffix="]" text-case="capitalize-first"/>
-          <group delimiter=": ">
-            <text term="available at" text-case="capitalize-first"/>
-            <text variable="URL"/>
-          </group>
+          <choose>
+            <if type="motion_picture">
+              <text value="Online Video" prefix="[" suffix="]"/>
```

Comment:
This is as per the IEEE guide, but it's not going to be translatable. I'm not familiar with setting up terms for localization yet, so someone clue-ier than I could fix this up to make it localizable.